backend/package.json

```diff
@@ -26,8 +27,10 @@
   "devDependencies": {
     "@types/node": "^16.9.6",
     "@types/validator": "^13.6.3",
+    "jest": "^27.2.4",
     "nodemon": "^2.0.13",
     "prisma": "^3.1.1",
+    "supertest": "^6.1.6",
```

Comment:
Okay, probably makes more sense to get to examples for this when we actually have a working API endpoint, but idk how to use this personally